fastlane/swift/ArgumentProcessor.swift

```diff
@@ -64,6 +65,15 @@ struct ArgumentProcessor {
         } else {
             self.commandTimeout = SocketClient.defaultCommandTimeoutSeconds
         }
+        
+        var laneParameters = [String: String]()
+        for arg in fastlaneArgs {
+            let lowercasedName = arg.name.lowercased()
+            if lowercasedName != "timeoutseconds" && lowercasedName != "lane" && lowercasedName != "logmode" {
```

Comment:
I think the intent of this could be made more clear by using `.filter()` instead of a for loop